Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -2001,6 +2002,7 @@ def main():
     handle_proxy()
     cmd = demisto.command()
     LOG(f'command is {cmd}')
+    demisto.log(BASE_URL)
```

Comment:
To can fetch it in the UT
![Screen Shot 2021-08-12 at 13 45 15](https://user-images.githubusercontent.com/78307768/129184172-e52e80c6-8deb-4f18-a9d5-1ae970a279d2.png)

Reply:
This is not the way to do that.
You should find a way to do the UT without adding code that is not relevant for the integration.

Reply:
I fixed it. It's simpler than I thought